box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/SharedLinkAccessFragment.java

```diff
@@ -55,7 +49,24 @@
 
     private View mPasswordHeader;
 
-    
+    private boolean mPasswordProtectedLinksSupported = false;
```

Comment:
can we set the default here to true? I'd rather a free user try it and fail rather than an enterprise user that should have the feature not be able to use it.